spec/spec_helper.rb

```diff
@@ -19,9 +18,7 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
-# Checks for pending migrations before tests are run.
-# If you are not using ActiveRecord, you can remove this line.
-ActiveRecord::Migration.check_pending! if defined?(ActiveRecord::Migration)
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
so this change seems unrelated - but I guess if everything is green we shouldn't worry - but might be nice to know where this crept in from - was it an explicit change you wanted? If so, is there a blog post or rails doc on the difference?  I see it mentioned here: https://relishapp.com/rspec/rspec-rails/docs/upgrade#pending-migration-checks  looks like a good change to make - ideally in separate PR perhaps, but it sounds like a clear win :-)